src/app/data-fetchers/tequilapi-proposal-fetcher.js

```diff
@@ -18,24 +18,38 @@
 // @flow
 
 import ProposalDTO from 'mysterium-tequilapi/lib/dto/proposal'
+import ProposalsQuery from 'mysterium-tequilapi/lib/adapters/proposals-query'
 import type { TequilapiClient } from 'mysterium-tequilapi/lib/client'
 import { FunctionLooper } from '../../libraries/function-looper'
 import type { Subscriber } from '../../libraries/publisher'
 import Publisher from '../../libraries/publisher'
 import type { ProposalFetcher } from './proposal-fetcher'
 
+const proposalsQueryWithMetric = new ProposalsQuery({ fetchConnectCounts: true })
+const filterFailedProposals = (proposal: ProposalDTO) => {
```

Comment:
`filter` is just one use-case - it's hard to understand what `true` and `false` return values mean:
```suggestion
const isProposalSuccessful = (proposal: ProposalDTO): boolean => {
```

Reply:
fixed